lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,13 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
+        if len(team.users) >= course_module.teams_configuration['max_team_size']:
```

Comment:
Is there a more django-ish way to do this using count(), or is len(team.users) the correct approach here?